### PR TITLE
Deploy GitHub pages

### PR DESCRIPTION
**Pull request by pgroves**
Add the command `npm run deploy-pages`, which deploys the files in dist/phylodiver-viz as a static site to github pages. See updates to the README for details of how it works.

Also adds some instructions for building this repo on an Ubuntu machine.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,28 @@
...
+## Prepping an Ubuntu VM:
+Prepping local ubuntu machine to build phylodiver-viz repo
+
+install npm, then angular using npm
+    https://angular.io/guide/setup-local
+
+1. `sudo apt-get install npm`
+2. `sudo npm install -g @angular/cli` 
+
+install an lts version of node
```
Comment (matthewberry) [inline README.md]: fwiw, I expect that if you install node first, it'll install npm for you

File: README.md [hunk 2]
```diff
@@ -2,6 +2,28 @@
...
+    https://github.com/nodesource/distributions/blob/master/README.md
+
+3. `curl -fsSL https://deb.nodesource.com/setup_lts.x > setup_lts.sh`
+4. `sudo -E bash setup_lts.sh`
+5. `sudo apt-get install -y nodejs`
+
+6. `ng version` #if this succeeds, angular cli is working
+
+install remaining project specific packages
+7. `npm install`
```
Comment (matthewberry) [inline README.md]: might want to mention you have to do this in the project top-level directory

**Review by matthewberry — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (pgroves) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (pgroves) [reply README.md]: fixed

**Pull request merged**